contracts/LoanCore.sol

```diff
@@ -281,7 +284,7 @@ contract LoanCore is
     }
 
     /**
-     * @notice Roll over a loan, atomically closing one and re-opening a new one with the
+     * @notice Rollover a loan, atomically closing one and re-opening a new one with the
```

Comment:
I think grammatically the separate words makes sense here. "Rollover" = noun, "roll over" = verb